apps/client/src/app/_components/Card/MainStoreCard.tsx

```diff
@@ -0,0 +1,13 @@
+import { Center } from "@chakra-ui/react";
+import type { ProductType } from "~/mocks/Apis/product/productApi";
+import { Card } from "./_components/Card";
+
+export const MainStoreCard = ({ data }: { data: ProductType[] }) => {
+  return (
+    <Center position="absolute" top="0" flexDir="column" w="100%" height="auto" padding="40px 0 48px">
+      {data.map((it) => {
+        return <Card data={it} />;
+      })}
```

Comment:
it라는 것도 어떤 걸 의미하는지 파악하기 어려운 것 같아요 ㅠㅠ !

그리고 아래와 같이 return 키워드 없이 작성하는 게 좋아보입니다. 또 key prop도 필요해보입니다 👍
```suggestion
      {data.map((it) => <Card data={it} />)}
```

Reply:
오~ 고민해보겠습니다 !  data를 구체화 시키면 it도 추상적이지 않게 작성이 가능하겠어요 bb 네 ! return 이 버릇이 되다보니 ㅎㅎ 짧고 간단한 return의 경우, return을 안쓰는게 낫다고 이해하면 좋을까요?

Reply:
return 키워드 지우고 콜백인자는 store로 변경했습니다 !

key prop은 사실 msw 반복기 통해서 응답되는 storeIdx가 동일한 값들이 있어서...
우선 key prop에 storeIdx + Math.ramdom() 이렇게 임시용으로 작성하여 반영 해두었습니다 !~